src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml.cs

```diff
@@ -248,10 +259,13 @@ private void SetComboboxCurrentVersion(string comboboxText, IEnumerable<NuGetVer
             for (int i = 0; i < _versions.Items.Count; i++)
             {
                 DisplayVersion currentItem = _versions.Items[i] as DisplayVersion;
-                if (currentItem != null && (comboboxText == _versions.Items[i].ToString() || _versions.Items[i].ToString() == matchVersion?.ToString()))
+                if (currentItem != null &&
+                    _versions.Items[i] != null && // null separator
```

Comment:
What'd does this comment is trying to say? I wonder if we could be a bit more specific/descriptive.

Reply:
We use "null" in the UI to represent the bar that separates Latest/Installed versions with the rest of versions. 
![image](https://github.com/NuGet/NuGet.Client/assets/43253759/98d29a06-9686-4586-ac3c-063595c00d37)